pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -419,9 +419,10 @@ class Build {
         // Sign and archive jobs if needed
 
         def targets = ["windows"]
-
-        // Note: mac binaries for openj9 variant are signed by 3rd party
-        if (buildConfig.VARIANT != "openj9") {
+        if (buildConfig.VARIANT.equals("openj9")) {
```

Comment:
I think there needs to be a sign flag in the spec files. Future change for being able to upstream some of this.